### PR TITLE
Can create and see all of my course

### PR DESCRIPTION
**Pull request by Heffiros**
# Purpose
Need to manage (create, update) a course

# Install
python manage.py migrate

# Validation protocol
- [ ] Connect with a teacher and go to /course/ => You will see ''Pas de classe"
- [ ] Go to /course/create => Fill the form you are redirect to /course and you see your new course

### CODE REVIEW

File: app/migrations/0008_auto_20170522_2157.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    operations = [
+        migrations.RenameField(
+            model_name='usercourse',
+            old_name='teacher_id',
+            new_name='user_id',
+        ),
+        migrations.AddField(
+            model_name='course',
+            name='user_id',
```
Comment (vendethiel) [inline app/migrations/0008_auto_20170522_2157.py]: Is that the teacher_id?

Comment (Heffiros) [inline app/migrations/0008_auto_20170522_2157.py]: yes i rename it

Comment (vendethiel) [inline app/migrations/0008_auto_20170522_2157.py]: I thought we said we didn't need it anymore. We can just query e.g.

```
SELECT c.*
FROM courses c
INNER JOIN course_user cu ON cu.course_id = c.course_id
WHERE cu.user_id = ${userID}
```

File: app/models.py [hunk 2]
```diff
@@ -3,15 +3,14 @@
 
 class Course(models.Model):
     name = models.CharField(max_length=30)
-    user_id = models.ForeignKey(User)
-    
+
 class Profil(models.Model):
    user = models.OneToOneField(User)
    role = models.IntegerField()
```
Comment (vendethiel) [inline app/models.py]: please add `courses = models.OneToManyField(User)` (or something like that)

https://stackoverflow.com/questions/4443190/djangos-manytomany-relationship-with-additional-fields

**Pull request merged**